packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.tsx

```diff
@@ -184,7 +185,8 @@ export class DetailsRow extends React.Component<IDetailsRowProps, IDetailsRowSta
         data-automationid='DetailsRow'
         style={ { minWidth: viewport ? viewport.width : 0 } }
         aria-selected={ isSelected }
-        >
+        onContextMenu={ this._onItemContextMenu.bind(this, this.props.item, this.props.itemIndex) }
```

Comment:
I thought about that, but noticed that the DetailsHeader was doing it in this way as well:

onContextMenu={ this._onColumnContextMenu.bind(this, column) }

However, I guess that re-creating for the header is not nearly as bad as for every row.

Unless there is a reason to keep this consistency, I will put the bind in the constructor instead - or possibly use the @autobind decorator if that is the preferred pattern.